src/pudl/workspace/datastore.py

```diff
@@ -534,15 +534,15 @@ def _get_pudl_in(args: dict) -> Path:
     if args.pudl_in:
         return Path(args.pudl_in)
```

Comment:
If we could drop support for `--pudl_in`, we could purely rely on `get_defaults()` and drop this function altogether. I think that could make sense.

Reply:
IIRC the use-case for setting a different input directory is/was being able to test the downloads of archives from Zenodo independent of the user's working cache directory.

As you've already noticed... there's a ton of room for simplification in the settings system.